CHANGELOG.md

```diff
@@ -18,6 +18,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Adds helper `associateExecutionWithGranule` to `@cumulus/api-client/granules`
 - **CUMULUS-2583**
   - Adds `queued` as option for granule's `status` field
+- **CUMULUS-2630**
+  - Adds a new workflow `DiscoverGranulesBackground` that discovers and writes
```

Comment:
Nit: to example development project

What's the purpose of adding this?   To facilitate dev testing, or for users?  If the latter, should we have a PR from 2630 to the template/docs/???

Reply:
I answered in reverse order.  It's for dev testing.  But IDK if it would be useful for users.

Reply:
For users, this is the documentation on how to use the feature in question:

https://nasa.github.io/cumulus/docs/data-cookbooks/throttling-queued-executions

We don't have a full example workflow in our template repo for it or anything though. Whether we should is a fair question